formatTest/typeCheckedTests/input/attributes.re

```diff
@@ -360,6 +360,10 @@ external readFileSync2 : (
 /* Ensure that attributes on extensions are printed */
 [@test [@attr] [%%extension]];
 
+external debounce : (int, [@bs.meth] unit) => unit;
+
+external debounce : (int, [@bs.meth] unit) => unit = "debounce";
+
 external debounce : (int, [@bs.meth] unit) => unit = "";
```

Comment:
maybe not `.rei` files, I might have looked too hastily. Why are these still in the old format?

Reply:
I thought we want to be backward compatible, so I've kept the old style there for those "input" files.

Reply:
but the output still has the empty string (`""`) or am I reading it wrong?

Reply:
oh I think I brainfarted... Apologies!